apps/openmw/mwworld/cellstore.hpp

```diff
@@ -357,6 +362,12 @@ namespace MWWorld
         return mWeapons;
     }
 
+    template<>
+    inline CellRefList<ESM::Door>& CellStore::getReadOnly<ESM::Door>()
```

Comment:
No, that makes more sense.  I adjusted the functions around the older code's door loop, and forgot to change that.